src/main/java/com/snowflake/kafka/connector/internal/SnowflakeConnectionServiceV1.java

```diff
@@ -552,18 +582,22 @@ private void appendColumnsToTable(
       logColumn.append(columnName).append(" (").append(columnInfosMap.get(columnName)).append(")");
     }
 
+    executeStatement(tableName, appendColumnQuery.toString());
+
+    logColumn.insert(0, "Following columns created for table {}:\n").append("]");
```

Comment:
I know that you tried to fit into the code that already exists, but I doubt if it makes sense to construct log message in a such complicated way at all. We log ALTER TABLE statement in `executeQuery` and that's all we need.